docs/src/componentDocs/Drawer/playground/PlaygroundPage.tsx

```diff
@@ -0,0 +1,193 @@
+        required: false,
+        category: 'Optional Props',
+    },
+
+    // Shared Props
+    ...sharedPropsConfig,
+
+    // Other Configuration
+    // NONE
+];
+
+const DrawerPreview: PreviewComponent = ({ data }) => {
+    const { collapseIcon, expandIcon, ...rest } = data as unknown as Omit<
+        DrawerProps,
+        'collapseIcon' | 'expandIcon'
+    > & {
+        collapseIcon: string;
+        expandIcon: string;
+    };
+    const { updateData } = usePlaygroundValues();
+    const containerRef = useRef(null);
+    const getCollapseIcon = (value: string): any => {
+        switch (value) {
+            case 'Remove':
+                return { family: 'material', name: 'remove' };
+            case 'undefined':
+            default:
+                return undefined;
+        }
+    };
+    const getExpandIcon = (value: string): any => {
+        switch (value) {
+            case 'Add':
+                return { family: 'material', name: 'add' };
+            case 'undefined':
+            default:
+                return undefined;
+        }
+    };
+    const navGroupItems: NavItem[] = [
+        {
+            itemID: 'Overview',
+            title: 'Overview',
+            // onPress: (): void => updateData('activeItem', 'Overview'),
+            icon: { name: 'dashboard' },
+            // expandIcon: {family: 'material', name: 'arrow-drop-down'},
+            collapseIcon: getCollapseIcon(collapseIcon as unknown as string),
+            expandIcon: getExpandIcon(expandIcon as unknown as string),
+            items: [
+                {
+                    itemID: 'Monthly Report',
+                    title: 'Monthly Report',
+                    onPress: (): void => updateData('activeItem', 'Monthly Report'),
+                },
+                {
+                    itemID: 'Annual Report',
+                    title: 'Annual Report',
+                    onPress: (): void => updateData('activeItem', 'Annual Report'),
+                },
+            ],
+        },
+        {
+            itemID: 'Timeline',
+            title: 'Timeline',
+            onPress: (): void => updateData('activeItem', 'Timeline'),
+            icon: { name: 'toc' },
+        },
+        {
+            title: 'Devices',
+            itemID: 'Devices',
+            subtitle: '5 new warnings',
+            onPress: (): void => updateData('activeItem', 'Devices'),
+            icon: { name: 'devices' },
+        },
+        {
+            itemID: 'Schedule',
+            title: 'Schedule',
+            onPress: (): void => updateData('activeItem', 'Schedule'),
+            icon: { name: 'airport-shuttle' },
+        },
+    ];
+
+    return (
+        <Box
+            sx={{
+                m: '16px 0',
+                maxHeight: '100%',
+                maxWidth: '80%',
+                marginLeft: '10%',
+                position: 'relative',
+            }}
+            ref={containerRef}
+        >
+            <Drawer {...rest}>
```

Comment:
can we import { DRAWER_WIDTH } from '../../../utils' and use style={{ width: DRAWER_WIDTH, margin: 'auto' }} here?